app/mono_pkg.yaml

```diff
@@ -6,7 +6,7 @@ stages:
   - smoke_test:
     - group:
         - dartfmt
```

Comment:
I find these group entry names confusing, as they still use the format commands.

Can we use more general terms such as 'formatting', 'analysis', and 'tests' ?

Reply:
need to update mono_repo for that.

Remember, this was based on the travis syntax invented by natalie...~5 years ago